server/controllers/interviewController.js

```diff
@@ -1,22 +1,24 @@
+const mongoose = require('mongoose');
 const interviewModel = require('../models/Interview');
+const Interview = mongoose.model('Interview');
 
 const { secretKey } = process.env;
 const jwt = require('jsonwebtoken');
 const { User } = require('../models/User.js');
 
-function getUserInterviews(req, res) {
-  const id = jwt.verify(req.cookies.token, secretKey).user._id;
+// function getUserInterviews(req, res) {
+//   const id = jwt.verify(req.cookies.token, secretKey).user._id;
 
-  //Return user interviews here
-  User.findById(id).then((data) => {
-    if (data.error) {
-      res.status(400).json({ error: data.error });
-    }
-    res.status(200).json({ response: data.interviews });
-  });
-}
+//   //Return user interviews here
+//   User.findById(id).then((data) => {
+//     if (data.error) {
+//       res.status(400).json({ error: data.error });
+//     }
+//     res.status(200).json({ response: data.interviews });
+//   });
+// }
 
-module.exports = { getUserInterviews };
+// module.exports = { getUserInterviews };
```

Comment:
You probably can delete these comments altogether - if you want to look at this code again you can use github on an old commit